src/components/views/elements/AppTile.tsx

```diff
@@ -400,6 +400,16 @@ export default class AppTile extends React.Component<IProps, IState> {
             { target: '_blank', href: this.sgWidget.popoutUrl, rel: 'noreferrer noopener' }).click();
     };
 
+    private onMaxMinWidgetClick = (): void => {
+        // TODO-video make this not interfere with the new video conference design
+
```

Comment:
This note feels like going a bit too far... If it is an issue, I would expect it to be clear in testing, so I think we can remove this note.